worker/src/handlers/schedule.ts

```diff
@@ -1,181 +1,172 @@
 // Scheduled taks handler to manage the KV store
 import { CurrentAnalytics } from "../../../site/src/data";
-import { SanitizedPayload } from "../data";
+import {
+  baseQueueData,
+  bumpValue,
+  Queue,
+  QueueData,
+  SanitizedPayload,
+} from "../data";
 import { average } from "../utils/average";
-import { formatDate } from "../utils/date";
 
 export async function handleSchedule(event: ScheduledEvent): Promise<void> {
-  const core_analytics: Record<string, any> = {};
-  const storedAnalytics =
-    (await KV.get<{ [key: string]: CurrentAnalytics }>(
-      "core_analytics",
-      "json"
-    )) || {};
-  const storedData = await listStoredData();
-  console.log(JSON.stringify(storedData));
-  //const currentDataset: CurrentAnalytics = generateCurrentDataset(storedData);
-
-  const currentDate = formatDate(new Date());
-  const currentDateObj = new Date(
-    currentDate.year,
-    currentDate.month,
-    currentDate.day,
-    currentDate.hour
-  );
-  const timestampString = String(currentDateObj.getTime());
-  //const storeKey = `history:${timestampString}`;
+  await prosessQueue();
+}
+
+export async function prosessQueue(): Promise<void> {
+  const queue = (await KV.get<Queue>("queue", "json")) || {
+    entries: [],
+    data: baseQueueData,
+  };
 
-  const dataKeys = Object.keys(storedAnalytics);
-  const lastDataEntry = storedAnalytics[dataKeys[dataKeys.length - 1]];
+  if (!queue.data || Object.keys(queue.data).length === 0) {
+    queue.data = baseQueueData;
+  }
 
-  for (const key of dataKeys) {
-    core_analytics[key] = {
-      active_installations: storedAnalytics[key].active_installations,
-      installation_types: storedAnalytics[key].installation_types,
-    };
+  if (queue.entries.length === 0) {
+    // No entries, get list
+    console.log("No entries, get list");
+    queue.entries = await listKV("uuid");
   }
 
-  core_analytics[timestampString] = {
-    ...lastDataEntry,
-    active_installations: storedData,
-  };
+  // Prosess the first 850 entries in the array
+  for (const entryKey of queue.entries.slice(0, 850)) {
```

Comment:
Could also use `splice` here 🤷‍♂️

Reply:
Do we rather miss data or have duplicate data?

Reply:
Oh we don't store anyway when we fail... so it would not matter...

Reply:
Yeah, I like that, will adjust

Reply:
nevermind